tests/integration/postgres.spec.ts

```diff
@@ -0,0 +1,286 @@
+import { expect } from "chai";
+
+import * as dfapi from "df/api";
+import * as dbadapters from "df/api/dbadapters";
+import * as adapters from "df/core/adapters";
+import { RedshiftAdapter } from "df/core/adapters/redshift";
+import { dataform } from "df/protos/ts";
+import { suite, test } from "df/testing";
+import { compile, getTableRows, keyBy } from "df/tests/integration/utils";
+import { PostgresFixture } from "df/tools/postgres/postgres_fixture";
+
+suite("@dataform/integration/postgres", { parallel: true }, ({ before, after }) => {
+  let dbadapter: dbadapters.IDbAdapter;
+
+  const postgres = new PostgresFixture(5432, before, after);
+
+  before("create adapter", async () => {
+    dbadapter = await dbadapters.create(
+      {
+        username: "postgres",
+        databaseName: "postgres",
+        password: "password",
+        port: 5432,
+        host: PostgresFixture.host
+      },
+      "postgres",
+      { disableSslForTestsOnly: true }
+    );
+  });
+
+  test("run", { timeout: 60000 }, async () => {
+    const compiledGraph = await compile("tests/integration/postgres_project", "project_e2e");
+
+    const adapter = adapters.create(compiledGraph.projectConfig, compiledGraph.dataformCoreVersion);
+
+    // Redshift transactions are giving us headaches here. Drop tables sequentially.
```

Comment:
I guess changes like this are happening in the next PR?

Reply:
yup, that is the plan